cord/orm/project.py

```diff
@@ -93,3 +95,55 @@ class ProjectCopyOptions(Enum):
     COLLABORATORS = 'collaborators'
     DATASETS = 'datasets'
     MODELS = 'models'
+
+
+# copy pasted class from api-server
+class StringEnum(Enum):
```

Comment:
This class is interesting. 

How is it different from accessing via:
Enum.name
Enum.value
Enum['value']
?

Reply:
Good point about `enum_instance.value`! I would still keep the class just for the `from_string` member. It allows doing e.g. `ReviewMode.from_string("reviewed")`